pkg/config/config_template.yaml

```diff
@@ -253,8 +253,9 @@ api_key:
 # might change depending on the OS.
 # dogstatsd_so_rcvbuf:
 #
-# Additional tags to append to all metrics received by this server. Useful for
-# tagging all metrics reporting from a single host without resorting to host tags.
+# Additional tags to append to all metrics, events and service checks received by
```

Comment:
Could we mention dogstatsd somewhere ?

Reply:
updated, with something a bit shorter (`dogstatsd metrics`)